### DISCUSSION TITLE
Replacing and/or updating parameter database

### DISCUSSION THREAD

**Discussion by LStarck** [equinor/neqsim#818] [use of NeqSim] [answered]
I'm not sure whether I've encountered a bug, or if there is something about how to run NeqSim that I've misunderstood. I'd like to flash a fluid that has additional (pseudo-)components to, and slightly different critical values for some of the components that are defined in the neqsim COMP.csv file.

Consequently I have made my own .csv file, but am not able to replace the neqsim 'COMP' table, due to errors with the jNeqSim.util.database.NeqSimDataBase.replaceTable and jNeqSim.util.database.NeqSimDataBase.updateTable functions as used in [this notebook](https://colab.research.google.com/github/EvenSol/NeqSim-Colab/blob/master/notebooks/PVT/parameter_database.ipynb).

**To Reproduce**

1. Go to [the notebook](https://colab.research.google.com/github/EvenSol/NeqSim-Colab/blob/master/notebooks/PVT/parameter_database.ipynb) 
2. Run cells.
3. The following errors will appear below different cells.
Error 1: AtrributeError: type object 'neqsim.util.database.NeqSimDataBase' has no attribute 'replaceTable'
Error 2: java.lang.RuntimeException: java.lang.RuntimeException: neqsim.util.exception.InvalidInputException: NeqSimDataBase:updateTable - Input path - Resource /content/COMPupdatedparam.csv not found

The same errors appear in my local set up, regardless of experimentation with relative and absolute input paths.

**Questions**
Are the functions I'm trying to apply deprecated? 
Are there other ways to create new components?

**Comment by EvenSol** [comment, marked as answer]
Good observation. This is a bug that has been fixed in #811 and #812. We will make a new release, and it should work.

**Reply by LStarck** [reply]
Thank you! I'll keep an eye out for the new release on pypi :)

**Reply by EvenSol** [reply]
bug fixed in release 2.5.6.
